angular.json

```
@@ -174,20 +174,28 @@
             "inlineStyleLanguage": "scss",
             "assets": [
               "projects/cardiovascular-health/src/favicon.ico",
-              "projects/cardiovascular-health/src/assets"
+              "projects/cardiovascular-health/src/assets",
+              "projects/cardiovascular-health/src/launch.html"
             ],
             "styles": [
-              "projects/cardiovascular-health/src/styles.scss"
+              "./node_modules/@angular/material/prebuilt-themes/indigo-pink.css",
+              "./dist/ngx-charts-on-fhir/themes/default.scss"
```

it looks like you replaced the wrong stylesheet here.

```suggestion
              "./dist/ngx-charts-on-fhir/themes/default.scss",
              "projects/cardiovascular-health/src/styles.scss"
```